### DISCUSSION TITLE
Input field should be directly editable

### DISCUSSION THREAD

**Discussion by jordanade** [mdbassit/Coloris#164] [Ideas]
The colorpicker is wonderful, but it prevents the user from directly editing the native input field's value. Instead, a seemingly redundant input field is included within the dropdown. It would more intuitive to simply allow the native input field to be used to enter or paste hex values.

Thanks!

**Comment by mdbassit** [comment]
You can edit the native input field by closing the color picker using the `ESC` key or the close button. 

This is the desired behavior of this color picker and it's unlikely to change because of my own personal and professional uses.
